.github/workflows/pr-e2e.yml

```
@@ -93,6 +93,17 @@ jobs:
           gh pr checkout ${{ needs.triage.outputs.pr_num }}
           git checkout ${{ needs.triage.outputs.commit_sha }}
 
+      - name: Run regex checks
+        env:
+          COMMENT_BODY: ${{ github.event.comment.body }}
+        run: |
+          MESSAGE="$COMMENT_BODY"
+          REGEX='/run-e2e (.+)'
+          if [[ "$MESSAGE" =~ $REGEX ]]
+          then
+            export E2E_TEST_REGEX="$(echo ${BASH_REMATCH[1]} | head -1)"
+          fi
```

We need to call to `e2e-regex-check`
```suggestion
      - name: Run regex checks
        id: regex-validation
        env:
          COMMENT_BODY: ${{ github.event.comment.body }}
        run: |
          MESSAGE="$COMMENT_BODY"
          REGEX='/run-e2e (.+)'
          if [[ "$MESSAGE" =~ $REGEX ]]
          then
            export E2E_TEST_REGEX="$(echo ${BASH_REMATCH[1]} | head -1)"
          fi
          make e2e-regex-check
```

---

Yes, forgot to add that.